src/app/ad-authoring/ad-authoring.component.html

```diff
@@ -3,17 +3,22 @@
     <mat-expansion-panel>
         <mat-expansion-panel-header>
             <mat-panel-title>
-                Text
+                Call To Action
             </mat-panel-title>
         </mat-expansion-panel-header>
 
         <mat-form-field>
             <mat-label>Landing Page URL</mat-label>
             <input matInput 
-                type="text" 
-                placeholder="Landing Page URL" 
-                [ngModel] = "landingUrl"
-                (ngModelChange)="updateLandingUrl($event)"/>
+                   type="text" 
+                   placeholder="https://example.com" 
+                   [ngModel] = "landingPageUrl"
+                   (ngModelChange)="updateLandingUrl($event)"
```

Comment:
nit: updateLandingPageUrl to keep it consistent?

Reply:
Changed everything to landingUrl